documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_1_Security_Perspective/4_1_2_Identity_and_Trust_Management.md

```diff
@@ -0,0 +1,115 @@
+
+## Trust Bootstrapping and Trust Chains
+Identifying and authenticating an IDS connector requires an evaluation of many complementary aspects. The following table provides an overview of these aspects and the entities responsible for them.
+
+| High-Level Aspect | Detailed Aspect | Trust Level | Entity Responsible | Validation necessary for issuing proof | Proof the IDS connector needs to validate |
+| --- | --- | --- | --- | --- | --- |
+| Connector ID (C_UID) belongs to communication partner | C_UID is unique | all | CA | Before issuing Connector Identity Certificate, process depends on implementation of C_UID | Connector Identity Certificate |
+| | Mapping of C_UID to Connector Instance Key (CIK) | all | CA | - | Connector Identity Certificate, usage of key material by communication partner |
+| | Only the connector has access to CIK | 1 | Operator | - | - |
+| | | 2, 3 | CA and Hardware Manufacturer | CA verifies hardware protection for CIK (Trust chain to HW Manufacturer) | Connector Identity Certificate |
+| C_UID is used by certified connector stack | - | 1 | Operator | **No validation**, operator must only request certificates for instances of certified SW stacks | - |
+| | Platform ID (P_UID) is unique | 2, 3 | CA | Before issuing connector identity certificate, process depends on implementation of P_UID | Platform Identity Certificate |
+| | Mapping of P_UID to Platform Instance Key (PIK) | 2, 3 | CA | - | Platform Identity Certificate |
+| | Only the identified platform has access to PIK | 2, 3 | CA and Hardware Manufacturer | CA verifies hardware protection for PIK (Trust chain to HW Manufacturer) | Platform Identity Certificate |
+| | Mapping C_UID to P_UID | 2, 3 | CA | Before issuing Connector Identity Certificate, CA ensures that the CIK is protected by the PIK | Mapping to P_UID in Connector Identity Certificate |
+| | Measurements for currently running software stack | 2, 3 | HW Trust Anchor, Certified Software | - | Measurements signed by PIK for platform and optionally service instance (in case of 1:1 mapping), Measurements signed by CIK for service instance (in case of multiple services on one platform) |
+| | Measurements belong to certified software stack | 2, 3 | Evaluation Facility and Certification Body | Part of Certification Process | Signed SW Manifests, Provided Measurements |
+| Connector with C_UID is operated by certified organization | C_UID belongs to component managed by a certified organization | all | CA | Before issuing Connector Identity Certificate, the CA validated that requesting organization has IDS Operational Environment Certification. | Connector Identity Certificate |
```

Comment:
A provider/consumer not operating a Connector shall not be required to be certified (if IDS is terminated in the Connector, i.e. policies can be completely enforced within the connector => just forwarding data to consumer without policies and maybe without IDS-messages). 
In case every provider or consumer, even though they do not operate a connector, need a certification, IDS will NOT be adopted by SMEs (IMHO). Such SMEs cannot or are most probably are not willing to afford a certification. Especially startups would never been able to participate in IDS-related data sharing, as some k€ might already be too much.

Reply:
I am totally with you that certification is too expensive for SMEs. But somehow you have to build additional trust with unknown partners. 

Just to make it clear:
- If only the operator is certified, the data must not leave the operator, i.e. the data must not be forwarded to the data consumer, because otherwise the data would be transfered to an uncertified and therefore untrusted partner. In my opinion, forwarding data (via data transfer or via user interface) to the data consumer makes up a large part of the use cases. 
- Data sovereignty is only certified from Trust Level 2, i.e. to get data sovereignty certified you have to have a hardware anchor, because you don't trust the operator, who is the only one who is certified....

Reply:
However, my understanding of the relationship between data consumer and operator of the connector is not that the operator only uses a connector as some kind of proxy to then transfer all the queried data to the consumer. 
Instead my understanding would be that the data consumer is always just a (human) user of the connector and its applications but the data is intended to remain stored and processed on the device operated by the operator and, thus, in a certified environment.
A transfer of data from the connector (under the operator's control) to a backend/device under the control of a data consumer must only be allowed under consideration of the usage control policies, i.e. if any usage control policies restrict the access, usage or transfer of data, then the data must not leave the connector (unless on the other side is a connector with usage control enforcement as well).

Reply:
For Trust Level 1, the data sovereignty (in form of adherence with usage control restrictions) is not technically enforced and, thus, not part of the certification for the devices/components. Nonetheless, legal or operational measures may be taken to agree on policies and adhere to them when processing or transferring data. Higher Trust Levels are intended to add technical means for fulfilling defined usage control restrictions to prevent violations of the agreed rules at the data consumer side on accident (level 2) or on purpose (level 3).